app/views/manage_outcomes/outcomes/_outcomes.html.erb

```diff
@@ -13,7 +13,13 @@
         <td><%= render "progress_bar",
           amount: outcome.active_assessments_with_results_count,
           total: outcome.active_assessments_count %></td>
-        <td><%= link_to t(".edit_outcome"), edit_manage_outcomes_outcome_path(outcome) %></td>
+        <td>
+          <%= link_to t(".edit_outcome"), edit_manage_outcomes_outcome_path(outcome) %>
+
+          <% if outcome.outcome_assessments.any? %>
+            <%= link_to t(".view_assessments"), manage_assessments_course_assessments_path(@course, outcome_ids: outcome) %>
```

Comment:
No, it's not really necessary in this context. I should update the form not to send it when filtering on the assessments screen as well.